packages/template-retail-react-app/app/components/reset-password/reset-password-success.jsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {Fragment} from 'react'
+import PropTypes from 'prop-types'
+import {FormattedMessage} from 'react-intl'
+import {Alert, Button, Stack, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {AlertIcon, BrandLogo} from '@salesforce/retail-react-app/app/components/icons'
+import {noop} from '@salesforce/retail-react-app/app/utils/utils'
+import ResetPasswordFields from '@salesforce/retail-react-app/app/components/forms/reset-password-fields'
+
+const PasswordResetSuccess = () => (
```

Comment:
I don't think this component is being used and instead the code is directly in reset-password/index.jsx

Reply:
I think this is a new component we are adding for this feature? If we are not using it, it is best to remove to avoid redundant code